stan/math/prim/fun/gp_matern32_cov.hpp

```diff
@@ -66,8 +65,8 @@ gp_matern32_cov(const std::vector<T_x> &x, const T_s &sigma,
   check_positive_finite(function, "length scale", length_scale);
 
   T_s sigma_sq = square(sigma);
-  T_l root_3_inv_l = sqrt(3.0) / length_scale;
-  T_l neg_root_3_inv_l = -1.0 * sqrt(3.0) / length_scale;
+  T_l root_3_inv_l = std::sqrt(3.0) / length_scale;
+  T_l neg_root_3_inv_l = -1.0 * std::sqrt(3.0) / length_scale;
```

Comment:
This should just be `T_l neg_root_3_inv_l = -root_3_inv_l`, and similarly for the other places in this and other `gp_matern` files.

Reply:
Oh yeah, good call. Didnt look at this hard enough to see that. Was just trying to fix the issue at hand.

Reply:
Fixed in https://github.com/stan-dev/math/pull/1702/commits/652bed1fc4032658bed2271095d34b22b2eb041d, the neg_ version is only used in one place so I ommited the extra local variable.